blocks/subscriptions-block/components/OrderInformation/index.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+
+import { usePhrases, Link, Heading, Paragraph, Stack } from "@wpmedia/arc-themes-components";
+import { FeatureDetails } from "../OfferCard";
+import currency from "../../utils/currency";
+
+const OrderSummary = ({ orderDetails, className }) => {
```

Comment:
The [ticket](https://arcpublishing.atlassian.net/browse/ASUB-8301) says this component should take 
a. orderDetail
    b. paymentType: ‘stripeIntents’ || ‘paypal’ || undefined
    c. campaignName
    d. offerURL
    e. showOfferURL (boolean)
    f. showPriceDescription (boolean)
   g. ShowProductFeatures (boolean)
   Do we not need these as props anymore?